adafruit_character_lcd/character_lcd.py

```diff
@@ -483,7 +498,7 @@ def create_char(self, location, pattern):
         for i in range(8):
             self._write8(pattern[i], char_mode=True)
 
-    def _write8(self, value, char_mode=False):
+    def _write8(self, value: [int], char_mode: bool = False) -> None:
```

Comment:
Since function arguments are defined at runtime, we should use immutable types, so for `value` it should just be `int`.

Reply:
I've just noticed that in the comment below we have `:param value: bytes` so should it be `Union[bytes, int]` ?

Reply:
No, a `bytes` object won't work here based on the use of `>>`

Reply:
Good point well made! Thx